tools/scoring/tests/dimensions/entity_identification_test.py

```diff
@@ -18,73 +18,32 @@
 
 from score.dimensions.entity_identification import EntityIdentification
 from score.constants import FileTypes, DimensionCategories
-from score.parse_config import ParseConfig
 
-from validate import handler as validator
-from validate.generate_universe import BuildUniverse
+from ..helper import TestHelper
```

Comment:
please use the path as we transform the imports when we bring the code internally.